src/main/pg/rx.c

```diff
@@ -20,7 +20,7 @@
 
 #include "platform.h"
 
-#if defined(USE_PWM) || defined(USE_PPM) || defined(USE_SERIAL_RX) || defined(USE_RX_MSP)
+#if defined(USE_PWM) || defined(USE_PPM) || defined(USE_SERIAL_RX) || defined(USE_RX_MSP) || defined(USE_RX_NRF24)
```

Comment:
Probably better make this `defined(USE_RX_SPI)` to capture the other SPI RX as well.

Reply:
done